python/trace-python.c

```diff
@@ -889,30 +860,32 @@ static struct uftrace_python_symbol *convert_function_addr(PyObject *frame, PyOb
 		Py_INCREF(code);
 	}
 
+	if (is_pyfunc)
+		func_name = get_python_funcname(frame, code, &is_main);
+	else
+		func_name = get_c_funcname(frame, code);
+
+	if (func_name == NULL)
+		return NULL;
+
+	/* code is not used anymore */
+	Py_DECREF(code);
```

Comment:
Please move this above the `if` statement so that we can release the `code` object even if `func_name` is NULL.

Reply:
fixed